app/Services/ReportbackService.php

```diff
@@ -14,14 +14,14 @@ class ReportbackService
     protected $reportbackRepository;
 
     /*
-     * Instance of \Rogue\Services\Phoenix\Phoenix
+     * Instance of \Rogue\Jobs\SendReportbackToPhoenix
      */
-    protected $phoenix;
+    protected $sendReportbackToPhoenix;
 
-    public function __construct(ReportbackRepository $reportbackRepository, Phoenix $phoenix)
+    public function __construct(ReportbackRepository $reportbackRepository, SendReportbackToPhoenix $sendReportbackToPhoenix)
     {
         $this->reportbackRepository = $reportbackRepository;
-        $this->phoenix = $phoenix;
+        $this->sendReportbackToPhoenix = $sendReportbackToPhoenix;
```

Comment:
Just got around to looking at this now, so sorry to comment after merge. 😁 It looks like this injected class and property aren't used anywhere? Dispatching by creating a new instance inline (like you did below) is pretty standard convention so this should be safe to remove.